src/Analyzers/Microsoft.Analyzers.Extra/DiagDescriptors.cs

```diff
@@ -64,14 +64,6 @@ internal static class DiagDescriptors
         description: Resources.ArraysDescription,
         defaultSeverity: DiagnosticSeverity.Warning);
 
-    public static DiagnosticDescriptor EnumStrings { get; } = Make(
-        id: "EA0006",
```

Comment:
ditto

The change also needs to be reflected in docs/list-of-diagnostics.md.

Reply:
Same here, would you mind logging an issue to clean things up if they weren't from this?